client/locales/de.json

```diff
@@ -0,0 +1,402 @@
+            "balanced": "Ausgeglichen",
+            "turbo": "Turbo",
+            "max": "Maximal",
+            "mop": "Wischen"
+        }
+    },
+    "map": {
+        "status": "Status",
+        "battery": "Batterie",
+        "connecting": "Verbinden...",
+        "statusText": "{{status, lowercase}}",
+        "errorText": "{{error, lowercase}}",
+        "confirmFullCleaning": "Zonenreinigung kann nicht gestartet werden: Keine Zonen spezifiziert. Soll stattdessen eine komplette Reinigung gestartet werden?",
+        "cannotResume": "Nichts zur Wiederaufnahme!",
+        "tooManyZones": "Es können nicht mehr als 5 Zonen auf der Karte gesetzt werden."
+    },
+    "zones": {
+        "headerZones": "Eingestellte Zonen",
+        "headerGotoLocations": "Eingestellte Zielpositionen",
+        "headerForbiddenMarkers": "Verbotsmarkierungen",
+        "noZonesConfigured": "Bisher wurden keine Zonen eingestellt.",
+        "noSpotsConfigured": "Bisher wurden keine Zielpunkte eingestellt.",
+        "editForbiddenZones": "Einstellen von Verbotszonen",
+        "noPersistentData": "Dauerhafte Daten sind abgeschaltet oder nicht verfügbar. Es können keine Verbotsmarkierungen bearbeitet werden.",
+        "addButton": "Hinzufügen",
+        "renameButton": "Umbenennen",
+        "deleteButton": "Löschen",
+        "newZoneName": "Zonenname spezifizieren ...",
+        "newSpotName": "Markierungsname spezifizieren...",
+        "enterZoneName": "Bitte einen Zonenamen eingeben",
+        "enterSpotName": "Bitte einen Markierungsnamen eingeben",
+        "renameZone": "Name für neue Zone \"{{zone}}\":",
+        "renameSpot": "Name für neue Markierung \"{{spot}}\":",
+        "confirmDeleteSpot": "Soll die Markierung \"{{spot}}\" wirklich gelöscht werden?",
+        "confirmDeleteZone": "Soll die Zone \"{{zone}}\" wirklich gelöscht werden?",
+        "zonesPageBack": "Zonen",
+        "zoneEditTitle": "Bearbeiten von Zone: {{zone}}",
+        "tooManySubzones": "Es können nicht mehr al 5 Unterzonen in einer Zone gesetzt werden.",
+        "tooManyMarkers": "Es können keine weitere Verbotsmarker gesetzt werden!",
+        "zonesSavedOK": "Zone erfolgreich gespeichert!",
+        "spotsPageBack": "Markierung",
+        "spotEditTitle": "Bearbeiten von Markierung: {{spot}}",
+        "spotsSavedOK": "Markierung erfolgreich gespeichert!",
+        "forbiddenMarkersPageBack": "Verbotsmarkierungen",
+        "forbiddenMarkersTitle": "Markierungen bearbeiten",
+        "forbiddenMarkersSavedOK": "Verbotsmarkierungen erfolgreich gespeichert!"
+    },
+    "control": {
+        "startButton": "Manuelle Kontrolle starten",
+        "stopButton": "Manuelle Kontrolle beenden"
+    },
+    "settings": {
+        "title": "Einstellungen",
+        "infoTitle": "Informationen",
+        "infoDesc": "Geräteinformationen anzeigen",
+        "timersTitle": "Zeitsteuerung",
+        "timersDesc": "Einstellen der Zeitabhängigen Steuerung",
```

Comment:
"timersDesc": "Timer und Zeitzoneneinstellungen",
would fit better to the content